### DISCUSSION TITLE
Autocomplete off

### DISCUSSION THREAD

**Discussion by goshatravin** [react-hook-form/react-hook-form#2838] [Q&A]
Hi guys, there is a way how can i switch off autocomplete  ? Currently when i go to signUp page  i dont want to see autocomplete, i already try to add autocomplete="off" to the form and to the input, chrome ignore it

**Comment by bluebill1049** [comment]
`autoComplete`

**Reply by MannanAnxari** [reply]
what the syntax of ??

**Reply by elthundercloud** [reply]
This does not work and isn't a valid solution for most browsers.
